pkg/networkutils/network.go

```diff
@@ -224,13 +225,38 @@ func findPrimaryInterfaceName(primaryMAC string) (string, error) {
 	return "", errors.New("no primary interface found")
 }
 
+func (n *linuxNetwork) enableIPv6() (err error) {
+	//Make sure IPv6 is enabled on host interfaces
+	if err = n.procSys.Set(fmt.Sprintf("net/ipv6/conf/all/disable_ipv6"), "0"); err != nil {
+		if !os.IsNotExist(err) {
+			return errors.Wrapf(err, "setupVeth network: failed to enable IPv6 on hostVeth interface")
+		}
+	}
+	//Make sure IPv6 forwarding is enabled on all host interfaces
+	if err = n.procSys.Set(fmt.Sprintf("net/ipv6/conf/all/forwarding"), "1"); err != nil {
+		if !os.IsNotExist(err) {
+			return errors.Wrapf(err, "setupVeth network: failed to enable IPv6 forwarding on all host interfaces")
+		}
+	}
+
+	//Let's disable ipv6 forwarding on eth0 (Primary ENI). We will direct external v6 traffic via GW
+	if err = n.procSys.Set(fmt.Sprintf("net/ipv6/conf/eth0/forwarding"), "0"); err != nil {
```

Comment:
True, if we want to enable forwarding (i.e.,) if `conf/all/forwarding` is set to `0` then setting it to `1` on a particular interface will not be of any use. But, it does allow to selectively disable `forwarding` on certain interfaces although `forwarding` is enabled for `all` interfaces. Above conf setting is disabling forwarding on `eth0` so that it can continue to honor RA packets. It does take effect. 

For reference: https://lists.debian.org/debian-ipv6/2011/05/msg00046.html (talks about disabling per interface SLAAC)

But, as discussed offline - I will instead set `accept_ra` to `2` on `eth0` so that we don't need to do anything specific w.r.t `forwarding` for `eth0`.